plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/names/NameEmbeddings.kt

```diff
@@ -37,7 +37,16 @@ fun CallableId.embedExtensionSetterName(type: TypeEmbedding): ScopedKotlinName =
     embedScopedWithType(type, ExtensionSetterKotlinName(callableName))
 
 fun CallableId.embedLocalPropertyName(): KotlinName = SimpleKotlinName(callableName)
-fun CallableId.embedMemberPropertyName(): ScopedKotlinName = embedScoped(MemberKotlinName(callableName))
+fun CallableId.embedMemberPropertyName(isPrivate: Boolean): ScopedKotlinName {
+    val id = classId ?: error("Embedding non-member property $callableName as a member.")
+    val className = ClassKotlinName(id.relativeClassName)
+    val packageName = packageName
```

Comment:
nit: unnecessary?

Reply:
It is still necessary for the private fields and regarding public fields it is necessary to leave package and class in order to obtain correct comparison behavior.